spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -47,23 +47,23 @@
         expect(json['live']).to eq true
       end
 
-      it 'includes the timeout key' do
-        timeout =  Time.zone.now + 10
+      it 'includes the timeout key', freeze_time: true do
+        timeout = Time.zone.now + 10
         controller.session[:session_expires_at] = timeout
         get :active
 
         json ||= JSON.parse(response.body)
 
-        expect(json['timeout'].to_datetime.to_i).to be_within(1).of(timeout.to_i)
```

Comment:
I'm increasingly finding `be_within` to be quite the code smell for these sorts of issues.